src/main/java/com/redhat/devtools/lsp4ij/client/LanguageClientImpl.java

```diff
@@ -249,7 +249,11 @@ protected Object findSettings(@Nullable String section) {
                 return config;
             }
             String[] sections = section.split("[.]");
-            return findSettings(sections, json);
+            Object settings = findSettings(sections, json);
```

Comment:
It would be nice to move this fix inside SettingsHelper and write tests with this usecase in https://github.com/redhat-developer/lsp4ij/blob/main/src/test/java/com/redhat/devtools/lsp4ij/client/SettingsHelperTest.java